docs/maps/index.rst

```diff
@@ -44,112 +44,28 @@ a map without reference to the underlying data representation (e.g. whether a
-cube with a pixel size proportional to the Fermi-LAT PSF:
-
-.. code-block:: python
-
-    import numpy as np
-    from gammapy.maps import Map, MapAxis
-    from astropy.coordinates import SkyCoord
-
-    position = SkyCoord(0.0, 5.0, frame='galactic', unit='deg')
-    energy_axis = MapAxis.from_bounds(100., 1E5, 12, interp='log', name='energy', unit='GeV')
-
-    binsz = np.sqrt((3.0*(energy_axis.center/100.)**-0.8)**2 + 0.1**2)
-
-    # Create a WCS Map
-    m_wcs = Map.create(binsz=binsz, map_type='wcs', skydir=position, width=10.0,
-                          axes=[energy_axis])
-
-    # Create a HPX Map
-    m_hpx = Map.create(binsz=binsz, map_type='hpx', skydir=position, width=10.0,
-                          axes=[energy_axis])
-
-.. _mapslicing:
-
-Indexing and Slicing
---------------------
-
-All map objects feature a `~Map.slice_by_idx()` method, which can be used to
-slice and index non-spatial axes of the map to create arbitrary sub-maps. The
-method accepts a `dict` specifying the axes name and correspoding integer index
-or `slice` objects. When indexing an axis with an integer the corresponding axes
-is dropped from the returned sub-map. To keep the axes (with length 1) in the
-returned sub-map use a `slice` object of length one. This behaviour is
-equivalent to regular numpy array indexing. The following example demonstrates
-the use of `~Map.slice_by_idx()` on a map with a time and energy axes:
-
-.. testcode::
-
-    import numpy as np
-    from gammapy.maps import Map, MapAxis
-    from astropy.coordinates import SkyCoord
-
-    position = SkyCoord(0.0, 5.0, frame='galactic', unit='deg')
-    energy_axis = MapAxis.from_bounds(100., 1E5, 12, interp='log', unit='GeV', name='energy')
-    time_axis = MapAxis.from_bounds(0., 12, 12, interp='lin', unit='h', name='time')
-
-    # Create a WCS Map
-    m_wcs = Map.create(binsz=0.02, map_type='wcs', skydir=position, width=10.0,
-                          axes=[energy_axis, time_axis])
+.. _node_types:
 
-    # index first image plane of the energy axes and third from the time axis
-    m_wcs.slice_by_idx({'energy': 0, 'time': 2})
+Differential and integral maps
+------------------------------
 
-    # index first image plane of the energy axes and keep time axis unchanged
-    m_wcs.slice_by_idx({'energy': 0})
+`gammapy.maps` supports both differential and integral maps, representing
+differential values at specific coordinates, or integral values within bins.
+This is achieved by specifying the ``node_type`` of a ``MapAxis``. Quantities
+defined at bin centers should have a node_type of "center", and quantities
+integrated in bins should have node_type of "edges". Interpolation is defined
+only for differential quantities.
 
-    # slice first three images of the energy axis at a fixed time
-    m_wcs.slice_by_idx({'energy': slice(0, 3), 'time': 0})
+For the specific case of the energy axis, conventionally, true energies are have
```

Comment:
I think having a distinction between true and reco energies is a very IACT specific thing. So it might be better to mention what these terms mean.

Reply:
In order to avoid definition repetitions we could just add permalink to the glossary entries and point to them when necessary

Reply:
yes, that is is a much better idea. 
My point was just that people should know what reco energy means...